counterexamples/transportation/segment/road/bad-road-flags-invalid-isPrivate-flag.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id": "invalidIsPrivateFlag",
+    "type": "Feature",
+    "geometry": {
+      "type": "LineString",
+      "coordinates": [[2, 2], [3, 3]]
+    },
+    "properties": {
+      "theme": "transportation",
+      "type": "segment",
+      "version": 19,
+      "updateTime": "2023-02-23T00:04:00-08:00",
+      "subType": "road",
+      "road": {
+        "flags": ["isPrivate"]
+      },
+      "extExpectedErrors": [
+        "[I#/properties/road] [S#/properties/properties/unevaluatedProperties] not allowed"
+      ]
```

Comment:
Hmmmm. Is it possible to do a bit better than this? Is there a line item that specifically relates to `flags`?

Reply:
I couldn't find anything related to flags as that was my first intention since Im not a fan of this generic error message too, but this the string that gets returned when the error occurs.

Reply:
Aren't the seventh and eighth lines in your screenshot (one of the two) what you're looking for?

```
[I#/properties/road/flags/0] [S#$defs/propertyDefinitions/road/properties/flags/items/oneOf/0/$ref] doesn't validate with '$defs/propertyDefinitions/roadFlag'
    [I#/properties/road/flags/0] [S#$defs/propertyDefinitions/roadFlag/enum] value must be one of "isBridge", "isLink", "isTunnel", "isUnderConstruction"
```

One of the two should work.

Reply:
Changed it to the second error.